src/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.csproj

```diff
@@ -2,10 +2,16 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AssemblyVersion>4.3.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.6</NuGetTargetMoniker>
+    <PackageTargetFramework>netstandard1.7</PackageTargetFramework>
```

Comment:
This shouldn't be needed, and in fact will most likely throw an error/warning on the build. With the new changes in master, this will be automatically infered based on your `<NuGetTargetMoniker>` value